packages/amazonq/.changes/next-release/Feature-09ac6888-de83-418b-afa7-d1cc58f687b2.json

```diff
@@ -0,0 +1,4 @@
+{
+	"type": "Feature",
+	"description": "Add grepSearch dependency for agentic chat"
```

Comment:
question: does this require a change log?

Reply:
after this PR will this feature be available? If so its worth it to mention. But i'd probably say something like `agentic chat: add grep tool` or something like that

Reply:
this PR will not enable the tool itself, this is a prerequisite step only (no functional change with this PR alone)

Reply:
you don't need changelog if you are just adding a dependency